wvdecrypter/wvdecrypter.cpp

```diff
@@ -1418,7 +1438,11 @@ class WVDecrypter : public SSD_DECRYPTER
   virtual void DestroySingleSampleDecrypter(AP4_CencSingleSampleDecrypter* decrypter) override
   {
     if (decrypter)
+    {
+      // close session before dispose
+      decrypter->CloseSessionId();
```

Comment:
you can leave untouched libbento (Ap4CommonEncryption.h)
by removing virtual to CloseSessionId method,
and replace this with:
`static_cast<WV_CencSingleSampleDecrypter*>(decrypter)->CloseSessionId();`

Reply:
Yes, it's just because of base class. Will change it